ci-operator/step-registry/cucushift/installer/check/vsphere/staticip/cucushift-installer-check-vsphere-staticip-commands.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+STATIC_IPS="${SHARED_DIR}"/static-ip-hosts.txt
+
+static_ips=$(yq-go r "$STATIC_IPS" 'hosts.*.networkDevice.ipAddrs.*' | cut -d "/" -f 1)
+node_ips=$(oc get nodes -o=jsonpath='{.items[*].status.addresses[?(@.type=="InternalIP")].address}')
+
+for node_ip in $node_ips; do
```

Comment:
If ip is not assigned correctly, from output message, seems unable to know on which node.
How about to loop each node name, then get node ip and check if it is in static_ips[@]?